2019_BOSC/abstract/abstract.tex

```diff
@@ -49,8 +49,8 @@
 While there were no major new modules introduced in Biopython 1.73
 (December 2018) or Biopython 1.74 (expected May/June 2019), there have
 been lots of incremental improvements.
-In terms of lines of code changed, a substantial proportion have been
-in-line documentation (Python docstrings), used to generate human readable
+In terms of lines of code changed, a substantial proportion has been
```

Comment:
"lines of code" (plural, thus "have") versus "substantial proportion" (tricky). Honestly I don't know which is correct, either reads OK to me.